src/Microsoft.Management.Deployment/PackageCatalogReference.cpp

```diff
@@ -78,7 +78,21 @@ namespace winrt::Microsoft::Management::Deployment::implementation
                 // Check if search behavior indicates that the caller does not want to do local correlation.
                 if (m_compositePackageCatalogOptions.CompositeSearchBehavior() != Microsoft::Management::Deployment::CompositeSearchBehavior::RemotePackagesFromRemoteCatalogs)
                 {
-                    ::AppInstaller::Repository::Source installedSource = ::AppInstaller::Repository::Source{ ::AppInstaller::Repository::PredefinedSource::Installed };
+                    ::AppInstaller::Repository::Source installedSource;
+                    auto manifestInstalledScope = GetManifestScope(m_compositePackageCatalogOptions.InstalledScope());
+                    if (manifestInstalledScope.first == ::AppInstaller::Manifest::ScopeEnum::User)

```

Comment:
seems like you can just assign manifestInstalledScope.first directly to the local variable.